site/content/docs/main/csi.md

```diff
@@ -38,11 +34,12 @@ See [Enabling Features][1] for more information about managing client-side featu
 This section documents some of the choices made during implementation of the Velero [CSI plugins][2]:
 
 1. VolumeSnapshots created by the Velero CSI plugins are retained only for the lifetime of the backup even if the `DeletionPolicy` on the VolumeSnapshotClass is set to `Retain`. To accomplish this, during deletion of the backup the prior to deleting the VolumeSnapshot, VolumeSnapshotContent object is patched to set its `DeletionPolicy` to `Delete`. Deleting the VolumeSnapshot object will result in cascade delete of the VolumeSnapshotContent and the snapshot in the storage provider.
-1. VolumeSnapshotContent objects created during a `velero backup` that are dangling, unbound to a VolumeSnapshot object, will be discovered, using labels, and deleted on backup deletion.
-1. The Velero CSI plugins, to backup CSI backed PVCs, will choose the VolumeSnapshotClass in the cluster that has the same driver name and also has the `velero.io/csi-volumesnapshot-class` label set on it, like
+2. VolumeSnapshotContent objects created during a `velero backup` that are dangling, unbound to a VolumeSnapshot object, will be discovered, using labels, and deleted on backup deletion.
+3. The Velero CSI plugins, to backup CSI backed PVCs, will choose the VolumeSnapshotClass in the cluster that has the same driver name and also has the `velero.io/csi-volumesnapshot-class` label set on it, like
 ```yaml
```

Comment:
We should be using `1.` for all the numbers, because that makes it easier to add or remove items from the list without having to re-number everything. The Markdown renderer can be a little picky thought about indentation within a list. If you indent the code sample here (1 tab i think), then the the numbers should be fine as `1.`

Reply:
My markdown editor made the change automatically, I will change it back